pubspec.yaml

```diff
@@ -1,12 +1,13 @@
 name: osm_nominatim
 description: A library to perform OSM Nominatim searches also supporting reverse searches
-version: 2.0.1
+version: 3.0.0
```

Comment:
Please don't bump the version

Reply:
Resolved in 79db7145d2f545df38a39d9bd89b17b6fd8c42e0.